services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java

```diff
@@ -304,13 +307,73 @@ public void testFindTasks() throws  EntityNotFoundException {
     assertEquals(newAssignee, taskCaptor.getValue().getAssignee());
 
     taskService.findTasks(newAssignee, newAssignee, 1);
-    verify(taskHandler, times(1)).findTasks(newAssignee,newAssignee,1);
+    verify(taskHandler, times(1)).findTasks(newAssignee,new ArrayList<>(),newAssignee,1);
 
     assertEquals(task.getId(), taskCaptor.getValue().getId());
 
+
```

Comment:
As the code modified by this fix is more in TaskStorage, this task should be in TaskTestStorage

Reply:
Done !!